### ISSUE TITLE
Support cursor rendering

### ISSUE THREAD

**Issue by hassan379** [ra1nty/DXcam#10]
Thank you for this library. it helps me a lot
can u please support cursor rendering 
Thanks again

**Comment by ra1nty** [comment]
Hi, thanks for trying it out. It's a good idea to add cursor support. I'm currently traveling and will definitely look into that after I got back.

**Comment by artem-levashov** [comment]
Hi. Do you have any progress with cursor  support?

**Comment by notwopr** [comment]
Yes this is also a feature I am very interested in.  Thank you for dxcam!

**Comment by newage99** [comment]
Also interested in this feature! ^^

**Comment by ra1nty** [comment]
Cursor capture is now supported through WinRT backend
